project_planning/project.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    type = fields.Selection(
+        [('ADM', 'ADM'),
+         ('DEV', 'DEV'),
+         ('FS', 'FS'),
+         ('PM', 'PM'),
+         ('QA', 'QA'),
+         ('SAL', 'SAL'),
+         ('TR', 'TR'),
+         ('TS', 'TS')],
+        string='Type', required=True, help="""
+        Type of task. Following values are available:
+            - ADM: administrative task
+            - DEV: development task
+            - FS: functional specification task
+            - PM: project managment task
+            - QA: quality control task
+            - SAL: sales task
+            - TR: training task
+            - TS: technical specification task
+        """)
+
+    wbs = fields.Char('WBS', help="Work Breakdown Structure")
```

Comment:
cc @duanyp1991 
Normally the wbs field is create by another module and we should not need to declare it here